### ISSUE TITLE
The accuracy is far lower than the accuracy mentioned in the paper

### ISSUE THREAD

**Issue by HHHendricks** [skumra/robotic-grasping#17]
I run train_network.py to train on the jacquard dataset, and the accuracy of the network test obtained is much lower than the accuracy in the paper. What is going on?
This is my parameter setting:
python train_network.py --dataset jacquard --dataset-path xxx --description train_jacquard

The accuracy tested on the Cornell Grasping Dataset is almost 0, while the accuracy tested on the Jacquard Dataset is only 75%.

**Comment by skumra** [comment]
Firstly, in the paper we report cross-validation accuracy on same datasets. This means that we split the same dataset for training and validation. Training on one dataset and testing on another dataset is not what we reported in the paper.

To help you reproduce the results, I recommend starting with the Cornell Grasping Dataset as the default parameters in the code as set for this dataset. Please try training and testing the model using the Cornell Grasping Dataset. We have also provided trained models if you just want to use the model for testing.

**Comment by HHHendricks** [comment]
But I still have a problem. I found that the accuracy of testing and training on the Cornell Dataset is basically the same as that in the paper; but the accuracy of training and testing on the Jacquard Dataset is still very low. The highest accuracy is only 80%, which is far less than the 94% mentioned in the paper. What is going on?

**Comment by skumra** [comment]
Just to confirm that I understand your issue correctly, you are able to reproduce the results reported in the paper using the Cornell Grasping Dataset with the default parameters. But you are unable to reproduce the results using the Jacquard Dataset with the default parameters. Is that correct?

**Comment by HHHendricks** [comment]
Yes, that's what I mean. I'm sorry, my English is not very good, so my expression is not very clear.

**Comment by skumra** [comment]
The default parameters are tuned for Cornell Grasping Dataset. You won't be to get the reported accuracy for Jacquard Dataset using the default parameters as this dataset is quite different from Cornell Grasping Dataset.

We have a journal paper in review that discusses the details of our design choices and parameter tuning for each dataset. I will share the journal paper once it is published. It will help you set the parameters to reproduce the results on Jacquard Dataset.

**Comment by HHHendricks** [comment]
I understand, thank you very much. Can you tell me how you set the parameters to achieve the accuracy in the paper or when will the new journal paper be published?

**Comment by HHHendricks** [comment]
Hello skumra, please tell me how to set the parameters if I want to train the network with my own dataset?

**Comment by skumra** [comment]
The journal paper should be available in next 5-6 months. 

The scope of this work was limited to training on two public datasets (Cornell and Jacquard) and testing them on real robot robot. Training the network on a customer dataset was not explored, but it is something that can be an extension of our work. You would need to tune the parameters to get a good success rate for your dataset. Parameters such as `batch-size` and `batches-per-epoch` depend on the size of your dataset.

**Comment by HHHendricks** [comment]
Okay, thank you very much! I wish you all the best in your research and hope to see your papers soon.

**Comment by skumra** [comment]
Thank you @HHHendricks !

**Comment by MJ0108** [comment]
Hello,thanks for your excellent work! I would like to ask whether the journal paper you mentioned earlier, which contains details discussing the design selection and parameter tuning of each database, has been published?  What's the title?  Thank you very much!

**Comment by MJ0108** [comment]
> The default parameters are tuned for Cornell Grasping Dataset. You won't be to get the reported accuracy for Jacquard Dataset using the default parameters as this dataset is quite different from Cornell Grasping Dataset.
> 
> We have a journal paper in review that discusses the details of our design choices and parameter tuning for each dataset. I will share the journal paper once it is published. It will help you set the parameters to reproduce the results on Jacquard Dataset.

Hello,thanks for your excellent work! May I ask whether the journal paper you mentioned earlier containing design details and parameter adjustments for each dataset has been published?  What's its subject?  thank you
